debian/control

```diff
@@ -0,0 +1,25 @@
+Source: one
+Section: devel
+Priority: extra
+Maintainer: Neural Network Acceleration Solution Developers <nnfw@samsung.com>
+Build-Depends: cmake, debhelper (>=9), dh-python, python3-all
+Standards-Version: 3.9.8
+Homepage: https://github.com/Samsung/ONE
+
+Package: one-compiler
+Architecture: any
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Description: on-device neural engine compiler
+
+Package: one-compiler-dev
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Can `one-compiler-dev` work without `one-compiler` ? If not, we need to add depend on `one-compiler`

Reply:
@seanshpark I think `one-compiler-dev` should work without `one-compiler`. I'll add `lib` to its install list.

Reply:
> I'll add lib to its install list.

Does this mean, both `one-compiler` and `one-compiler-dev` will install `lib` ?

Reply:
> Yes. That's what I meant to.

I think this may make conflict on `lib`.

Reply:
> I think `one-compiler-dev` should have a dependency on `one-compiler`.

Is it for preventing `one-compiler-dev` from having duplicate `lib` list?